src/components/__tests__/Map.test.tsx

```
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+import { Map } from '../Map';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)